MAUI/ListView/swiping.md

```diff
@@ -0,0 +1,628 @@
+
+To swipe an item indefinitely, set the `SfListView.SwipeOffset` property by considering the width or height of the SfListView with [SfListView.Orientation](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Orientation) accordingly.
+
+{% tabs %}
+{% highlight c# %}
+ListView.PropertyChanged += ListView_PropertyChanged;
+
+private void ListView_PropertyChanged(object sender, PropertyChangedEventArgs e)
+{
+  if (e.PropertyName == "Width" && ListView.Orientation == Orientation.Vertical && ListView.SwipeOffset != ListView.Width)
+     ListView.Offset = ListView.Width;
+  else if (e.PropertyName == "Height" && ListView.Orientation == Orientation.Horizontal && ListView.SwipeOffset != ListView.Height)
+     ListView.Offset = ListView.Height;
+}
+{% endhighlight %}
+{% endtabs %}
+
+Download the entire source code from GitHub `here`
+
+![Indefinite swipe in listview](Images/swiping/maui-listview-swipe-an-item-indefinetly.jpg)
+
+## How to edit data by swiping?
+
+The `SfListView` allows editing the item data using either `SfListView.EndSwipeTemplate` or `SfListView.StartSwipeTemplate` by loading edit view into the respective template after swiping the item.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+ <syncfusion:SfListView > 
+  <syncfusion:SfListView.EndSwipeTemplate> 
+      <DataTemplate x:Name="EndSwipeTemplate"> 
+        <Grid BackgroundColor="#DC595F" HorizontalOptions="Fill" VerticalOptions="Fill"> 
+          <Grid> 
+            <Label Grid.Row="0" 
+                    HeightRequest="50" 
+                    WidthRequest="50" 
+                    BackgroundColor="Transparent" 
+                    Text="EditItem"> 
+               <Grid.GestureRecognizers>
+                   <TapGestureRecognizer Tapped="TapGestureRecognizer_Tapped"/>
+               </Grid.GestureRecognizers>
+            </Label> 
+          </Grid> 
+        </Grid> 
+      </DataTemplate> 
+  </syncfusion:SfListView.EndSwipeTemplate> 
+ </syncfusion:SfListView>
+</ContentPage>
+{% endhighlight %} 
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+listView.EndSwipeTemplate = new DataTemplate(() =>
+{
+  var grid = new Grid()
+  {
+    BackgroundColor = Colors.FromHex("#009EDA"),
+    HorizontalOptions = LayoutOptions.Fill,
+    VerticalOptions = LayoutOptions.Fill
+  };
+
+  var grid1 = new Grid();
+  TapGestureRecognizer tapped = new TapGestureRecognizer();
+  grid1.GestureRecognizers.Add(tapped);
+  tapped.Tapped += Grid_Tapped;
+  var label = new Label()
+  {
+    HeightRequest =50,
+    WidthRequest = 50,
+    BackgroundColor =Colors.Transparent,
+    Text ="EditItem"
+  };
+
+  grid1.Children.Add(label);
+  grid.Children.Add(grid1);
+  return grid;
+});
+{% endhighlight%}
+{% endtabs %}
+
+To set tapped items binding context for pop-up page, follow the code example.
```

Comment:
To set tapped items binding context for the pop-up page, follow the code example below:

Reply:
resolved